### DISCUSSION TITLE
When is WSL2 coming to other cards like the 7800XT?

### DISCUSSION THREAD

**Discussion by ghost** [ROCm/ROCm#3672] [Q&A]
recently set this up for a friend that has a 7900GRE and it worked pretty well, any idea when this will come to other cards like the 7800XT?

**Comment by schung-amd** [comment]
Unfortunately, while extending official hardware support is on our radar, we're unable to provide any updates/commitments at this time regarding the 7800XT and other gfx1101 cards (and similarly for other currently unsupported architectures). Closing as a duplicate of https://github.com/ROCm/ROCm/discussions/2599 for the 7800XT specifically, please refer there for further discussion. I will update there when I'm able to provide more information.
